build.gradle

```diff
@@ -48,6 +48,11 @@ subprojects {
   }
 }
 
+task install {
```

Comment:
this fixes publishing to jitpack. this is probably a temporary solution - this essentially replicates the plugin that was used in the past that is deprecated to avoid having to mess with the build task (which attempts to build the whole project). I suspect eventually they will remove support for the install task